extension/src/experiments/model/index.ts

```diff
@@ -521,6 +518,12 @@ export class ExperimentsModel extends ModelWithPersistence {
     return this.availableBranchesToShow
   }
 
+  public getAvailableBranchesToSelect() {
+    return this.availableBranchesToShow.filter(
+      branch => branch !== this.currentBranch
```

Comment:
[Q] Should this be done in `setBranches` or further up the change in `collectBranches`?